ant-protocol/src/storage/chunks.rs

```diff
@@ -25,6 +25,9 @@ pub struct Chunk {
 }
 
 impl Chunk {
+    /// The default maximum size of a chunk is 1MB
+    pub const DEFAULT_MAX_SIZE: usize = 1024 * 1024;
```

Comment:
Made this `DEFAULT_MAX_SIZE` for now, we can/should review this once we have stabilized the size.